tp53 nf1 readme (#652)
AlexsLemonade/OpenPBTA-analysis: analyses/tp53_nf1_score/README.md

## Apply classifiers trained on TCGA RNA-seq data

> **Module author:** edited Greg Way @gwaygenomics' code by Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6)) Chante Bethell (@cbethell) and Jaclyn Taroni (@jaclyn-taroni)

Reviewer: ```suggestion
**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6)) and Jaclyn Taroni (@jaclyn-taroni); code adapted from Gregory Way ([@gwaygenomics](https://github.com/gwaygenomics))
```
Using https://github.com/AlexsLemonade/OpenPBTA-analysis/commits/master/analyses/tp53_nf1_score as a starting point and then looking at the commits that include scientific decision making.
